Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp

```diff
@@ -259,6 +260,16 @@ void MolDraw2DSVG::getStringSize(const std::string &label, double &label_width,
   }
 }
 
+namespace {
+void escape_xhtml(std::string &data) {
```

Comment:
Where do we use spirit? (At the airport and unable to search code)
It seems silly to add a dependency here for something so simple though